vllm/engine/arg_utils.py

```diff
@@ -168,6 +169,10 @@ def add_cli_args(
             help='load model sequentially in multiple batches, '
             'to avoid RAM OOM when using tensor '
             'parallel and large models')
+        parser.add_argument('--ray-workers-use-nsight',
+                            type=bool,
+                            default=False,
+                            help='use nsight to profile ray workers')
```

Comment:
Nit (just for consistency with other boolean flags):

```suggestion
        parser.add_argument('--ray-workers-use-nsight',
                            action='store_true',
                            help='If specified, use nsight to profile ray workers')
```